CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Winit Contributing Guidelines
+
+## Scope
+
+Winit aims to provide a generic platform abstracting the main graphic platforms (Windows, MacOS, x11, 
+Wayland, Android, iOS and the web platform via Emscripten).
+
+Most platform expose capabilities that cannot be meaningfully transposed to the others. Winit does not
+aim at supporting every single functionaly of every platform, but rather abstract the set of 
```

Comment:
s/aim at supporting/aim to support
s/functionaly/functionality
s/rather abstract/rather to abstract